modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1037,15 +1041,34 @@ EFAElement3D::createChild(const std::set<EFAElement *> & CrackTipElements,
           center += cut_plane_points[i];
         center /= cut_plane_points.size();
 
+        for (unsigned int i = 0; i < cut_plane_points.size(); ++i)
+        {
+          int iplus1 = i < cut_plane_points.size() - 1 ? i + 1 : 0;
+          int iplus2 = i + 2;
+          if (iplus2 + 2 - cut_plane_points.size() >= 0)
```

Comment:
This is causing a compile error that says this comparison is always true. I'm impressed that the compiler caught that! That's irrelevant if you delete this code block, though.